sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1313,6 +1315,7 @@ MemCpyCommand::MemCpyCommand(Requirement SrcReq,
   }
 
   MWorkerQueue = MQueue->is_host() ? MSrcQueue : MQueue;
+  MEvent->setWorkerQueue(MWorkerQueue);
```

Comment:
no, because default Command constructor sets MQueue as WorkerQueue. In these two accessors MQueue is not always valid choice so we need to overwrite it.